src/test/kotlin/org/objectionary/ddr/integration/resolver/ResolverBase.kt

```diff
@@ -145,18 +132,27 @@ open class ResolverBase : TestBase {
         return XSLDocument(stripXml).with(ClasspathSources()).transform(xmir)
     }
 
+    override fun deleteTempDir(pathToSource: Path) {
+        val tmpDir = File("${pathToSource}_$postfix")
```

Comment:
@c71n93 We are trying to avoid compound names. According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html